src/py21cmfast/src/IonisationBox.c

```diff
@@ -1392,6 +1420,29 @@ LOG_DEBUG("prev_min_density=%f, prev_max_density=%f, prev_overdense_small_min=%f
                 counter += 1;
         }
 
+
+        for (x=0; x<user_params->HII_DIM; x++){
+            for (y=0; y<user_params->HII_DIM; y++){
+                for (z=0; z<user_params->HII_DIM; z++){
+                    if ((box->z_re_box[HII_R_INDEX(x,y,z)]>0) && (box->xH_box[HII_R_INDEX(x,y,z)] < TINY)){
```

Comment:
one of these is sufficient no?

Reply:
> one of these is sufficient no?

I think I was worried about cells that recombine a lot. In this case, z_re_box>0 but xH_box > TINY. I intend to keep their temperature as returned by ComputePartiallyIoinizedTemperature